gitlab/resource_gitlab_service_pipelines_email.go

```diff
@@ -0,0 +1,114 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServicePipelinesEmail() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServicePipelinesEmailCreate,
+		Read:   resourceGitlabServicePipelinesEmailRead,
+		Update: resourceGitlabServicePipelinesEmailUpdate,
+		Delete: resourceGitlabServicePipelinesEmailDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabServicePipelinesEmailImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
I believe the `project` attribute should be `ForceNew: true`. Otherwise, if the user updates the attribute, it could add the service to the new project and leave the service enabled on the old project.